### PR TITLE
mega-linter-runner: Remove container by default, except of `no-remove-container` option is sent

### PR DESCRIPTION
**Pull request by nvuillam**
Fixes https://github.com/oxsecurity/megalinter/issues/3062

### CODE REVIEW

File: mega-linter-runner/lib/options.js [hunk 1]
```diff
@@ -137,7 +137,12 @@ export const optionsDefinition = optionator.default({
     {
       option: "remove-container",
       type: "Boolean",
-      description: "Remove MegaLinter Docker container when done",
+      description: "Remove MegaLinter Docker container when done (default: true since v7.8.0)",
+    },
+    {
+      option: "no-remove-container",
```
Comment (echoix) [inline mega-linter-runner/lib/options.js:143]: @nvuillam should this new option be set to be mutually exclusive like other sets below?

And shouldn’t release be mutually exclusive with image (you can’t add a tag to an already full name)?

Comment (nvuillam) [reply mega-linter-runner/lib/options.js:143]: @echoix indeed you are right :)

**Comment by huczas** [comment]
I have latest version 7.8.0, this feature is already merged but I'm not sure if it is working as expected?
I have worker image with preinstalled docker img megalinter (and many other stuff), it is used many times during day, should I add this "no-remove-container"?
For now nothing is deleted and docker container is not removed automatically from my workers - which is best for my case.

**Comment by echoix** [comment]
Removing the container is not the same thing as removing the image. The image pulled shouldn't be removed. Only the container started from that image. 

This will show you the containers on your system, including the ones that are stopped. Stopped containers can be restarted, like when you boot a machine a second time. 
```
docker ps --all
```

Usually, you'd probably want to have a new container from the same image, so that for one, it is clean, and that you can use different startup parameters (like env vars). Since you probably don't write the command manually, mega-linter-runner does, it shouldn't bother to remove the container after use. It is the equivalent of using the `--rm` in 
```
docker run --rm ...
```

**Pull request merged**